index.js

```diff
@@ -138,6 +139,10 @@ export const startEvaluate = async ({
     })
   })
 
+  // Update top measurement stations every 12 hours
+  const pgClient = await createPgClient()
+  setInterval(async () => { await updateTopMeasurementStations(pgClient) }, 1000 * 60 * 60 * 12)
```

Comment:
I'd like to discuss how much up-to-date we want to have this materialised view.

In the current proposal, we refresh the view every 12 hours. If the service process is started by Fly.io at 11.50 am UTC, we will refresh the view every day at 11.50 am UTC and 11.50 pm UTC. For most of the morning in Europe, the view will be showing data from the day before yesterday.

Is that good enough for this initial version? @juliangruber 

The alternative I am thinking of:
- At the process startup, if the time is between 0:00-0:30 am UTC, then run the update immediately.
- Always schedule the next update to be executed on the next day at 0:30 am UTC.

Thoughts?

Reply:
I believe this is good enough, let's iterate 👍

Reply:
ok 👍🏻